lang/system/pl.xml

```diff
@@ -164,22 +226,22 @@ Kod błędu: {}</an_error_occurred>
 			<days>30</days>
 			<days>31</days>
 		</small_mday>
-		<hidden_trophy>Skjult trophy</hidden_trophy>
-		<one_hour_ago>1 godz. temu</one_hour_ago>
-		<one_minute_ago>1 min temu</one_minute_ago>
-		<bronze>Bronse</bronze>
-		<gold>Gull</gold>
-		<platinum>Platina</platinum>
-		<silver>Sølv</silver>
+		<hidden_trophy>Ukryte trofeum</hidden_trophy>
```

Comment:
No change this, is fine already

Reply:
@Zangetsu38 In what language it was fine? XD

Reply:
His change is right

Reply:
Translate is from psvita directy

Reply:
I mean, croden and my typo
so keep this change